R/Rules-methods.R

```diff
@@ -3017,7 +3017,7 @@ setMethod("stopTrial",
       prob_target <- stopping@prob_target
 
       ## checks
-      stopifnot(is.probability(prob_target))
+      stopifnot(test_probability(prob_target))
```

Comment:
please use instead `assert_probability(prob_target)`, to avoid the `stopifnot`

Reply:
Updates completed, passed all checks